File: java/MASTER_SLAVE_WHOLE_PART_PROXY/DatosCliente.java
public class DatosCliente {
    //Atributos
    private String nombre;
    private String apellidoPaterno;
    private String apellidoMaterno;
    private int edad;
    private String tipoCliente;

    //Constructor
    public DatosCliente(){}

    //Métodos
    public void setDatosCliente(String nombre, String apellidoP, String apellidoM, int edad, String tipoC){
        this.nombre = nombre;
        this.apellidoPaterno = apellidoP;
        this.apellidoMaterno = apellidoM;
        this.edad = edad;
        this.tipoCliente = tipoC;
    }
    
    public String getDatosCliente(){
        return "\nDATOS DEL CLIENTE | NOMBRE: " + nombre + " " +  apellidoMaterno + " " + apellidoPaterno + " | EDAD: " + edad + " | TIPO DE CLIENTE: " + tipoCliente;
    }
}

File: php/MASTER_SLAVE_WHOLE_PART_PROXY/datos/DatosCliente.php
<?php


class DatosCliente{
    private  $nombre;
    private  $apellidoPaterno;
    private  $apellidoMaterno;
    private  $edad;
    private  $tipoCliente;

    public function setDatosCliente($nombre, $apellidoP, $apellidoM,$edad, $tipo){
        $this->nombre = $nombre;
        $this->apellidoPaterno = $apellidoP;
        $this->apellidoMaterno = $apellidoM;
        $this->edad = $edad;
        $this->tipoCliente = $tipo;
    }

    public function getDatosCliente(){
        $informacion = "<h2>Datos de Cliente</h2>";
        $informacion .= "Nombre: ".$this->nombre;
        $informacion .= "<br/>Apellido Paterno: ".$this->apellidoPaterno;
        $informacion .= "<br/>Apellido Materno: ".$this->apellidoMaterno;
        $informacion .= "<br/>Edad: ".$this->edad;
        $informacion .= "<br/>Tipo: ".$this->tipoCliente. '<br/>';

        return $informacion;   
    }
}

